protostar-rust/src/lib.rs

```diff
@@ -32,7 +26,7 @@ struct TestsFromFile {
 
 fn collect_tests_from_directory(
     input_path: &Utf8PathBuf,
-    linked_libraries: Option<&Vec<LinkedLibrary>>,
+    linked_libraries: &Option<Vec<LinkedLibrary>>,
```

Comment:
I think it is more idiomatic to do it the way it was before (here and in other places)
https://github.com/software-mansion/protostar/pull/2036#discussion_r1224144914

Reply:
ok fair

Reply:
[fdbck](https://github.com/software-mansion/protostar/pull/2059/commits/f83f77d95fc28d6557dbd04d98a83d3bce5938b9)